packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -85,11 +89,18 @@ class ImageCache {
     assert(value >= 0);
     if (value == maximumSize)
       return;
+    TimelineTask timelineTask;
+    if (!kReleaseMode) {
+       timelineTask = TimelineTask()..start('ImageCache.setMaximumSize', arguments: <String, dynamic>{'value': value});
```

Comment:
Nit: You might want to consider breaking this into multiple lines (below as well)

Reply:
Done